client/layout/index.jsx

```diff
@@ -155,7 +155,7 @@ class Layout extends Component {
 					isWooOAuth2Client( this.props.oauth2Client ) &&
 					this.props.wccomFrom,
 				'is-white-signup':
-					'signup' === this.props.sectionName &&
+					'signup' === this.props.sectionName && //TODO: Check flowName here and remove this comment
```

Comment:
You should be able to get the flow name using `getCurrentFlowName( state )` selector from `state/signup/flow/selectors`, and use that flow name an additional check here.

Reply:
Thanks @niranjan-uma-shankar for the tip! While working on this I realized that the `is-white-signup` class should be added to the `body` element and not as a section class. This has been fixed in https://github.com/Automattic/wp-calypso/pull/44644/commits/586f9dd4d77df91b922546aab23c55c6146990dc.